docs/content-server-api/pointerChanges/get.md

```diff
@@ -0,0 +1,76 @@
+# Get Pointer Changes
+
+List all deployment changes from the same pointer
```

Comment:
This description is kind of weird. It doesn't list all changes for the same pointer. It lists all changes made to pointers in general